tsl/src/remote/connection.c

```diff
@@ -1227,7 +1257,8 @@ remote_connection_open_with_options_nothrow(const char *node_name, List *connect
 					/* Should wait for write on socket */
 					events |= WL_SOCKET_WRITEABLE;
 					break;
-				default:
+				case PGRES_POLLING_ACTIVE:
+					Assert(false);
```

Comment:
This event is not used but kept for backwards compatibility according to libpq comment.